types/evse_manager.yaml

```diff
@@ -324,6 +324,39 @@ types:
       vendor_error:
         description: The error code of the vendor
         type: string
+  EnableDisableSource:
+    description: >-
+      Source of a Enable or Disable command/event
+    type: object
+    required:
+      - enable_source
+      - enable_state
+      - enable_priority
+    properties:
+      enable_source:
+        description: Specifies the source
+        type: string
+        enum:
+        - Unspecified
+        - LocalAPI
+        - LocalKeyLock
+        - ServiceTechnician
+        - RemoteKeyLock
+        - MobileApp
+        - FirmwareUpdate
+        - CSMS
+      enable_state: 
```

Comment:
Trailing whitespace

Reply:
more applicable to #505 where it will need to be addressed.